### PR TITLE
Update Espressif border router instructions.

### PR DESCRIPTION
**Pull request by zwx1995esp**

### CODE REVIEW

File: site/en/guides/border-router/espressif-esp32.md [hunk 1]
```diff
@@ -1,69 +1,73 @@
 # ESP Thread Border Router
 
-Contributor: https://github.com/gjc13
+Contributor: https://github.com/gjc13 https://github.com/zwx1995esp
```
Comment (Vyrastas) [inline site/en/guides/border-router/espressif-esp32.md:3]: Can you add yours on a separate line.  So there will be two individual contributor lines:

Contributor: https://github.com/gjc13
Contributor: https://github.com/zwx1995esp

Also, can you confirm you're affiliated with Espressif?  (seems so from your profile)  That way we can ensure the byline uses the Espressif logo on openthread.io.

File: site/en/guides/border-router/espressif-esp32.md [hunk 2]
```diff
@@ -1,69 +1,73 @@
...
+Clone the [esp-idf](https://github.com/espressif/esp-idf) and the [esp-thread-br](https://github.com/espressif/esp-thread-br) repository.
 
 ```
-$ cd ${IDF_PATH}/examples/openthread/ot_rcp
-$ idf.py --preview set-target esp32h2
-$ idf.py build flash
+$ git clone -b v5.1.1 --recursive https://github.com/espressif/esp-idf.git
+$ cd esp-idf
+$ ./install.sh
+$ . ./export.sh
```
Comment (Vyrastas) [inline site/en/guides/border-router/espressif-esp32.md:28]: Is this correct?  Or should it be

`$ ./export.sh`

Without the extra .

File: site/en/guides/border-router/espressif-esp32.md [hunk 3]
```diff
@@ -1,69 +1,73 @@
...
 
 ```
-$ cd ${IDF_PATH}/examples/openthread/ot_br
+$ cd esp-thread-br/examples/basic_thread_border_router
 $ idf.py menuconfig
 ```
 
 The network configuration items are:
 
+* Enable automatic start mode in Thread Border Router **ESP Thread Border Router Example > Enable the automatic start mode in Thread Border Router.**
```
Comment (Vyrastas) [inline site/en/guides/border-router/espressif-esp32.md:45]: Add a colon after "Router"

Enable automatic start mode in Thread Border Router: **ESP

File: site/en/guides/border-router/espressif-esp32.md [hunk 4]
```diff
@@ -1,69 +1,73 @@
...
 * Wi-Fi SSID and PSK: **Example Connection Configuration > connect using Wi-Fi interface**
-* Thread network parameters: **OpenThread network parameters**
+* Thread network parameters: **Component config > OpenThread > Thread Operational Dataset**
+
 
+## Step 3: Build and run the border router
 
-## Step 4: Build and run the border router
+Build the `esp-idf/examples/openthread/ot_rcp` example. The firmware doesn't need to be explicitly flashed to a device. It will be included in the Border Router firmware and flashed to the ESP32-H2 chip upon first boot(or the RCP firmware changed).
```
Comment (Vyrastas) [inline site/en/guides/border-router/espressif-esp32.md:52]: Missing space

boot(or --> boot (or

File: site/en/guides/border-router/espressif-esp32.md [hunk 5]
```diff
@@ -75,27 +79,12 @@ Done
 * IPv6 bidirectional connectivity.
 * SRP service registration and advertising proxy.
 * mDNS discovery proxy.
+* NAT64
+* Multicast forwarding
+* Web GUI based REST API.
+* OTA
```
Comment (Vyrastas) [inline site/en/guides/border-router/espressif-esp32.md:85]: The list formatting should be consistent, either all should end in periods, or all don't end in periods.

File: site/en/guides/border-router/espressif-esp32.md [hunk 6]
```diff
@@ -1,69 +1,73 @@
...
+Espressif provides an [ESP Border Router Board](https://www.espressif.com/en/news/Thread_Border_Router_Certification) which integrates the host SoC and the RCP into one module.
 
-## Hardware connection
-
-The ESP32-H2 RCP will be connected to the border router via UART.
+## Hardware platforms
 
 <figure>
-<a href="../../guides/images/otbr-esp-connection.jpg"><img src="../../guides/images/otbr-esp-connection.jpg" width="600" border="0" alt="ESP32-H2 UART" /></a><figcaption>ESP32-H2 UART connection</figcaption>
+<a href="../../guides/images/otbr-esp-br-board.png"><img src="../../guides/images/otbr-esp-br-board.png" width="600" border="0" alt="ESP Thread Border Router Board" /></a><figcaption>ESP Thread Border Router Board</figcaption>
```
Comment (Vyrastas) [inline site/en/guides/border-router/espressif-esp32.md:14]: Can you compress the new image a little bit.  It needs to be smaller than 1048576 bytes for importing into openthread.io

**Review by Vyrastas — CHANGES_REQUESTED** [review]
Thanks for updating this!  A few comments

File: site/en/guides/border-router/espressif-esp32.md (hunk 1 above)
Comment (zwx1995esp) [reply site/en/guides/border-router/espressif-esp32.md:3]: Yes, I am working for the Espressif.

File: site/en/guides/border-router/espressif-esp32.md (hunk 2 above)
Comment (zwx1995esp) [reply site/en/guides/border-router/espressif-esp32.md:28]: Yes, this extra dot is really needed, which you can refer to [ESP Programming Guide](https://docs.espressif.com/projects/esp-idf/en/stable/esp32s3/get-started/linux-macos-setup.html#step-4-set-up-the-environment-variables)

File: site/en/guides/border-router/espressif-esp32.md [hunk 7]
```diff
@@ -1,69 +1,73 @@
...
-Contributor: https://github.com/gjc13
+Contributor: https://github.com/gjc13 https://github.com/zwx1995esp
 
 ESP Thread border router is a FreeRTOS-based implementation running on a combination of Espressif's Wi-Fi and 802.15.4 SoCs.
 
 Hardware requirements:
 
-* An ESP32 series Wi-Fi dev kit. (ESP32, ESP32-C3, ESP32-S2 or ESP32-S3)
-* An [ESP32-H2](https://www.espressif.com/en/news/ESP32_H2) dev kit for running RCP.
+Espressif provides an [ESP Border Router Board](https://www.espressif.com/en/news/Thread_Border_Router_Certification) which integrates the host SoC and the RCP into one module.
```
Comment (chshu) [inline site/en/guides/border-router/espressif-esp32.md:9]:
```suggestion
Espressif provides an [ESP Border Router Board](https://www.espressif.com/en/news/Thread_Border_Router_Certification) which integrates the host SoC (ESP32-S3) and the RCP (ESP32-H2) into one module.
```

**Comment by zwx1995esp** [comment]
Hi @Vyrastas Could you please take a look again?

File: site/en/guides/border-router/espressif-esp32.md (hunk 6 above)
Comment (zwx1995esp) [reply site/en/guides/border-router/espressif-esp32.md]: Now the image is ~55000 bytes

**Review by Vyrastas — APPROVED** [review]
@zwx1995esp is there a name you'd like us to use for your contributor byline beyond "Zhangwx"?  I.e. like the "Jiacheng Guo" at the top here https://openthread.io/guides/border-router/espressif-esp32

**Comment by zwx1995esp** [comment]
> @zwx1995esp is there a name you'd like us to use for your contributor byline beyond "Zhangwx"? I.e. like the "Jiacheng Guo" at the top here https://openthread.io/guides/border-router/espressif-esp32

Yah, my name is Zhang Wenxu

**Comment by Vyrastas** [comment]
> > @zwx1995esp is there a name you'd like us to use for your contributor byline beyond "Zhangwx"? I.e. like the "Jiacheng Guo" at the top here https://openthread.io/guides/border-router/espressif-esp32
> 
> Yah, my name is Zhang Wenxu

Great I'll use that.  Thanks!

**Review by chshu — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**